sdk/communication/azure-communication-chat/CHANGELOG.md

```diff
@@ -1,15 +1,11 @@
 # Release History
 
-## 1.5.0-beta.1 (Unreleased)
+## 1.5.0 (2024-04-10)
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
-
+- Expanded `ChatAttachment` to include new enum `file` to enable communication users to recieve file attachments shared by Teams user in a Teams Interoperbility Chat.
+- Added support for a new communication identifier `MicrosoftTeamsAppIdentifierModel`.
```

Comment:
I don't see this added to the public models. It's only internally added and shouldn't be part of the changelog if it's not publicly added.

Reply:
thanks! will remove it from changelog